integration-tests/environments/electron/jenkins.groovy

```diff
@@ -0,0 +1,69 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2019 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+def onLinux() {
+  return {
+    node('docker') {
+      // Unstash the files in the repository
+      unstash 'source'
+      docker.build(
+        'ci/realm-js:electron',
+        // Based on node:10, installing xvbf
+        '-f integration-tests/environments/electron/Dockerfile .'
+      ).inside(
+        '-e HOME=/tmp' // NPM will create folders in ~/.npm
```

Comment:
The comment and the code doesn't seem to match?

Reply:
I think it does? ~/.npm will be /tmp/.npm to which it will have write access, which it wouldn't have had if `HOME` was not set.